src/errors/error.cr

```diff
@@ -4,7 +4,7 @@ module Mint
   class Error < Exception
     alias Value = String | Ast::Node | TypeChecker::Checkable |
```

Comment:
Like before `Tuple(Ast::Node, Array(Int32) | Int32)` is enough here since it's matching both cases.